components/Head.tsx

```diff
@@ -0,0 +1,79 @@
+import { FC } from 'react'
+import NextHead from 'next/head'
+
+type Props = {
+  ogImage?: string
+  title?: string
+  description?: string
+}
+
+/**
+ * Renders the <head> section of the HTML page, including meta tags, title, and favicon.
+ * @param ogImage - The URL of the Open Graph image to be displayed in social media shares.
+ * @param title - The title of the page.
+ * @param description - The description of the page.
+ */
+export const Head: FC<Props> = ({
+  ogImage = 'https://marketplace.reservoir.tools/og-image.png',
+  title = 'Reservoir Market | Open Source NFT Marketplace',
+  description = 'Reservoir Market is an open source NFT marketplace built with Reservoir.',
+}) => {
+  return (
+    <NextHead>
+      {/* CONFIGURABLE: You'll probably want to configure this all to have custom meta tags and title to fit your application */}
+
+      {/* Meta tags */}
+      <meta name="keywords" content="nft, ethereum, protocol" />
```

Comment:
Is it possible to just have the tags that are dynamic in here? E.g. keep the keywords, reservoir tags and anything else that's relatively static in the document page?

Reply:
Yes! I just didn't want to separate them because I wanted all the tags to be in the same spot.

I'll move them right now.

Reply:
Moved more tags to the global scope _document